### ISSUE TITLE
Teacher's mode inaccessible?

### ISSUE THREAD

**Issue by zuevval** [braille-systems/learn-braille#321]
Unable to enter teacher's code (keyboard buttons could not be pressed) with TalkBack

Samsung Galaxy S5 SM-G900P, Android 5.0

Accessibility services: TalkBack 12.1

`TODO` check other Android versions
